acceptance/lib/facter/acceptance/user_fact_utils.rb

```diff
@@ -42,6 +42,20 @@ def get_external_fact_script_extension(platform)
           '.sh'
         end
       end
+
+      # Retrieve the path to default location of facter.conf file.
+      #
+      def get_default_fact_dir(platform, version)
+        if platform =~ /windows/
+          if version < 6.0
+            File.join('C:', 'Documents and Settings', 'All Users', 'Application Data', 'PuppetLabs', 'facter')
```

Comment:
I did not have this case in the tests I wrote for the default location, and it currently passes all of our platforms.